docs/components/beacon/index.html

```diff
@@ -100,4 +99,67 @@ <h2 id="beacon-with-inline-styles">Beacon with Inline Styles</h2>
       </footer>
     </div>
   </section>
+
+  <section>
+    <header>
+      <h2 id="beacon-with-default-styles">Beacon with Color Override</h2>
+    </header>
+    <p>
+      This is an example implementation of <code>&lt;hx-beacon&gt;</code>
+      With color override.
+      Hexidecimal or named color values are valid, though hex values are
+      best practice for consistency.
+    </p>
+    <p class="hxSubdued hxSubBody">
+      <hx-icon type="exclamation-triangle"></hx-icon>
+        User can use <b>--hxBeacon</b> property for overriding the default Beacon
+        component color.
+        (Example:<b>--hxBeacon:#d32f2f or --hxBeacon:red</b>)
+    </p>
+    <div class="example" id="vue-beacon-default-styles" v-cloak>
+      <header>
+        <form class="beta-hxForm">
+          <fieldset>
+            <hx-text-control>
+              <input
+                id="txtBeaconDefaultStyles"
+                type="text"
+                v-model="beaconDefaultStyles"
+              />
+              <label for="txtBeaconDefaultStyles">
+                Beacon Inline Styles
+              </label>
+            </hx-text-control>
+            <p class="hxSubdued hxSubBody">
+              <hx-icon type="info-circle"></hx-icon>
+              Use percentages and/or pixel values to adjust the position
+              relative to the parent element.
+            </p>
+          </fieldset>
+        </form>
+      </header>
+
+      <div style="position: relative; border: 1px dashed black;"> <!-- demo snippet -->
+        <hx-beacon
+          @dismiss="onEvent"
+          :style="beaconDefaultStyles"
+        >
+        </hx-beacon>
+        <hx-disclosure class="hxBtn" aria-controls="basicMenuId">
+          <hx-icon type="cog"></hx-icon>
+          <span>Actions</span>
+          <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
+        </hx-disclosure>
+        <hx-menu id="basicMenuId" :open="isOpen">
```

Comment:
We already using `basicMenuId` on this page, so will need to use another _id_.

Please change to `overrideMenuId`.

Reply:
Changes have been done, Please review